ecosystem/sep-0002.md

```diff
@@ -20,9 +20,25 @@ For example: `bob*stellar.org`:
 
 * `bob` is the username,
 * `stellar.org` is the domain.
-The domain can be any valid RFC 1035 domain name. The username is limited to printable UTF-8 with whitespace and the following characters excluded: `*` ,`>` Although of course the domain administrator can place additional restrictions on usernames of its domain.
 
-Note that the `@` symbol is allowed in the username. This allows for using email addresses in the username of an address. For example: `maria@gmail.com*stellar.org`.
+### Username
+
+The username is limited to printable UTF-8 with whitespace and the following characters excluded: `*` ,`>`. Although of course the domain administrator can place additional restrictions on usernames of its domain.
+
+#### Email Addresses
+
+Email addresses may be used as the username as the `@` symbol is allowed in the username. This allows for using email addresses as the username of an address when the users username is their email address.
+
+For example: `maria@gmail.com*stellar.org`.
+
+#### Phone Numbers
+
+Phone numbers may be used as the username. When encoding a phone number as the username the ITU-T recommendations E.123 and E.164 and should be followed and that spaces be ommitted to ensure phone numbers are consistently encoded and do not collide when mixing international phone numbers.
+
+For example: `+14155550100`.
```

Comment:
The example phone number is part of the reserved set of numbers for fictional use in North America.

Reply:
should this example have the `*stellar.org` suffix?

Reply:
Thanks, fixed in ee77b3a.